### ISSUE TITLE
ConvexHullTraits_3::Less_distance_to_point_3 wrong description

### ISSUE THREAD

**Issue by mglisse** [CGAL/cgal#3421]
## Issue Details

Noticed in https://stackoverflow.com/q/52945100/1918193 .

https://doc.cgal.org/4.13/Convex_hull_3/classConvexHullTraits__3.html#a44e03ba1b3655d3f07bd45b30e1864e9 describes Less_distance_to_point_3 with a constructor that takes p and an operator() that takes q and r. This does not match the kernel concept 
https://doc.cgal.org/latest/Convex_hull_3/classConvexHullTraits__3.html and does not match reality, convex_hull_3.h clearly expects to build less_dist with no argument, and call less_dist(p,q,r).

## Environment

* CGAL version: 4.13

**Comment by sloriot** [comment]
Indeed the documentation is wrong, I'll fix it.

**Comment by mglisse** [comment]
In the description of Traits_xy_3, it says that it is calling ch_bykat, but that seems to depend on the iterator category. This matters because the concepts are not exactly the same.

**Comment by sloriot** [comment]
Indeed, from what I see `ch_akl_toussaint` should be called as the points are from an internal vector. I'll check with a real example and update the doc accordingly. Thanks!
